com.sleepfiles.OSCAR.yaml

```diff
@@ -0,0 +1,60 @@
+######################################################
+# Flatpak manifest for OSCAR
+# Erico Mendonca <erico.mendonca@gmail.com> Jan 2022 
+######################################################
+
+# Special OBS field because flatpak does not have a version field
+# Default will be '0' if the field is missing.
+#!BuildVersion: 1.3.1
+---
+app-id: com.sleepfiles.OSCAR
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: OSCAR
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home,xdg-run
+  - --device=dri
+build-options:
+  env:
+    - QMAKE_CXXFLAGS: -L/app/lib
+    - QMAKE_LDFLAGS: -L/app/lib
```

Comment:
as mentionned you don't need this.
```suggestion
```

Reply:
That's the problem: qmake won't find libGLU when linking if I don't specify it here.

Reply:
ok